src/components/MyKiva/BadgeContainer.vue

```diff
@@ -20,17 +20,6 @@
 				:src="solidComponent"
 			>
 		</div>
-		<div
-			v-if="isLocked"
-			class="tw-absolute tw-flex"
-			:class="{ 'animate-wiggle': animateLock }"
```

Comment:
Oh this is one thing we miss with using the image with the lock. Could you check with Nathan in the Slack thread if we are OK not having the animated lock? If so, looks like we can remove other code related to `animateLock`.

Reply:
Removed